mne/viz/raw.py

```diff
@@ -379,7 +385,7 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
     if show_options is True:
         _toggle_options(None, params)
     # initialize the first selection set
-    if order in ['selection', 'position']:
+    if order in ['selection', 'position', 'lasso']:
```

Comment:
You're using this set ('selection', 'position', 'lasso') at least 3x, maybe just initialize it as a list once and refer to it.